sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -268,6 +271,55 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
       });
   });
 
+  it("Should get application details when the current assessment is impacted and the trigger type is 'Related application changed'.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      appDataSource,
+      { student },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+        createSecondDisbursement: true,
+        currentAssessmentInitialValues: {
+          triggerType: AssessmentTriggerType.RelatedApplicationChanged,
+        },
+        firstDisbursementInitialValues: {
+          coeStatus: COEStatus.completed,
+        },
+      },
+    );
+    application.applicationStatus = ApplicationStatus.Completed;
+    application.currentAssessment.triggerType =
+      AssessmentTriggerType.RelatedApplicationChanged;
+    await applicationRepo.save(application);
```

Comment:
This part of code is not required anymore as it is handled by `saveFakeApplicationDisbursements`

![image](https://github.com/bcgov/SIMS/assets/54600590/55953f6f-498b-46c5-b076-098cb6f39ee8)